lib/main_publiccloud.pm

```diff
@@ -104,8 +114,7 @@ sub load_latest_publiccloud_tests {
     }
     elsif (get_var('PUBLIC_CLOUD_UPLOAD_IMG')) {
         loadtest "publiccloud/upload_image";
-    }
-    else {
+    } else {
```

Comment:
I like this, thank you! 😄

Reply:
`tools/tidy` did this I think :thinking:  But yes, I also prefer this style.